src/main/java/com/stories/domain/StoryDomain.java

```diff
@@ -60,8 +60,12 @@
 	
 	@ApiModelProperty(example="1", value="The history of the story")
 	private List<String> history;
+	
+	@ApiModelProperty(example="?", value="The tasks of the story")
```

Comment:
What do you mean with this character "?"?

Reply:
Sorry, I forgot to change that. As for the example, I see on other annotations from the same class that the value there is just "1", should I change it to that?

Reply:
You can change it for "New task"

Reply:
Thank you, it´s been already changed